app/config.go

```diff
@@ -632,7 +675,7 @@ func MakeCodec() {
 func Credentials() string {
 	bytePassword, err := terminal.ReadPassword(syscall.Stdin)
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
```

Comment:
Should likely stop execution or it will panic on the return statement
```suggestion
		log.Fatal(err)
```